pkg/virt-controller/watch/vmi.go

```diff
@@ -1210,6 +1209,17 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 			return nil
 		}
 
+		// Ensure the backend storage PVC is ready
+		var backendStoragePending bool
+		backendStoragePending, err = backendstorage.IsPVCPending(vmi, c.clientset)
```

Comment:
Maybe do it inline?
```suggestion
		backendStoragePending, err := backendstorage.IsPVCPending(vmi, c.clientset)
```

Reply:
What happens if PVC goes to `Lost` state?

Reply:
> Maybe do it inline?

`'err' redeclared in this block`

> What happens if PVC goes to `Lost` state?

Good point, changed to an error in that case